### PR TITLE
refs #207 Keeps window size

### PR DESCRIPTION
**Pull request by asmsuechan**
Hello, I dealt with https://github.com/BoostIO/Boostnote/issues/207

### CODE REVIEW

File: lib/main-window.js [hunk 1]
```diff
@@ -2,12 +2,18 @@ const electron = require('electron')
...
+const Config = require('electron-config')
+const config = new Config()
 
 var showMenu = process.platform !== 'win32'
+let windowSize = { width: 1080, height: 720 }
 
-var mainWindow = new BrowserWindow({
-  width: 1080,
-  height: 720,
+if(config.get('windowsize') === undefined) config.set('windowsize', windowSize)
```
Comment (sota1235) [inline lib/main-window.js]: Please add space after `if` statement. We compliant [JavaScript Standard Style](https://github.com/feross/standard)!

Comment (asmsuechan) [inline lib/main-window.js]: Thanks for your advice! I modify soon.

File: lib/main-window.js [hunk 2]
```diff
@@ -2,12 +2,18 @@ const electron = require('electron')
...
+windowSize = config.get('windowsize')
```
Comment (sota1235) [inline lib/main-window.js]: You can write the code more short.

```
const windowSize = config.get('windowsize') || { width: 1080, height: 720 }
```

File: lib/main-window.js [hunk 3]
```diff
@@ -2,12 +2,18 @@ const electron = require('electron')
...
+
+let mainWindow = new BrowserWindow({
```
Comment (sota1235) [inline lib/main-window.js]: Thank you for using `let` instead of `var` :D

Comment (asmsuechan) [inline lib/main-window.js]: 🙆

File: lib/main-window.js (hunk 2 above)
Comment (asmsuechan) [inline lib/main-window.js]: Wow absolutely true...

**Comment by asmsuechan** [comment]
I fixed it :pray:

**Comment by sota1235** [comment]
Thank you for fixing. Looks good to me!

**Pull request merged**